src/main/java/org/commonjava/indy/service/tracking/change/FoloTrackingListener.java

```diff
@@ -132,6 +137,11 @@ public void handleFileStorageEvent( final FileEvent event )
             return;
         }
         final AccessChannel accessChannel = AccessChannel.valueOf( (String) metadata.get( Constants.ACCESS_CHANNEL ) );
+        String keyString = event.getStoreKey();
+        if (!StringUtils.isNotBlank( keyString )) {
```

Comment:
Same

Reply:
updated in https://github.com/Commonjava/indy-tracking-service/pull/27